authority-portal-backend/authority-portal-quarkus/src/main/kotlin/de/sovity/authorityportal/web/pages/organizationmanagement/OrganizationInvitationApiService.kt

```diff
@@ -36,8 +36,18 @@ class OrganizationInvitationApiService(
 ) {
 
     fun inviteOrganization(invitationInformation: InviteOrganizationRequest, adminUserId: String): IdResponse {
+        userService.assertUserDoesNotExistInDbOrThrow(invitationInformation.userEmail)
```

Comment:
I would rename this function to 'userExistsInDb' and throw it on the service layer to make the code more readable. Otherwise it's implicit which exact exception will be thrown: 404, 409, 418

This is how CP handles the invitation conflicts:
![image](https://github.com/user-attachments/assets/408be85a-c9c6-4920-9f8f-2018d1ede586)

Tbh I'm not fully satisfied with this snippet, but generally I prefer to throw exception in the main business logic class and not somewhere else